.vscode/launch.json

```diff
@@ -29,9 +29,8 @@
             "name": "Launch RascalShell",
             "request": "launch",
             "mainClass": "org.rascalmpl.shell.RascalShell",
-            "vmArgs": "-ea",
```

Comment:
why are assertions removed?

Reply:
I needed the shell with bugfixes to run the compiler; with -ea it was way to slow. I'll put it back before merging.